### PR TITLE
Update tab config and add Demos section

### PR DESCRIPTION
**Pull request by Huanli-Meng**
### Motivation

Add a demo section to the doc.

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [ x ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: docs/lakehouse-sink.md [hunk 1]
```diff
@@ -268,7 +258,7 @@ You can create a configuration file (JSON or YAML) to set the properties if you
         "inputs": [
           "test-delta-pulsar"
         ],
-        "archive": "connectors/pulsar-io-lakehouse-{{connector:version}}.nar",
+        "archive": "connectors/pulsar-io-lakehouse-2.10.0.7.nar",
```
Comment (hangc0276) [inline docs/lakehouse-sink.md:261]: connectors/pulsar-io-lakehouse-2.10.0.7-cloud.nar

File: docs/lakehouse-sink.md [hunk 2]
```diff
@@ -156,7 +150,7 @@ You can create a configuration file (JSON or YAML) to set the properties if you
         "inputs": [
           "test-hudi-pulsar"
         ],
-        "archive": "connectors/pulsar-io-hudi-{{connector:version}}.nar",
+        "archive": "connectors/pulsar-io-hudi-2.10.0.7.nar",
```
Comment (hangc0276) [inline docs/lakehouse-sink.md:153]: connectors/pulsar-io-hudi-2.10.0.7-cloud.nar

File: docs/lakehouse-sink.md [hunk 3]
```diff
@@ -213,7 +205,7 @@ You can create a configuration file (JSON or YAML) to set the properties if you
         "inputs": [
           "test-iceberg-pulsar"
         ],
-        "archive": "connectors/pulsar-io-lakehouse-{{connector:version}}.nar",
+        "archive": "connectors/pulsar-io-lakehouse-2.10.0.7.nar",
```
Comment (hangc0276) [inline docs/lakehouse-sink.md:208]: `connectors/pulsar-io-lakehouse-{{connector:version}}-cloud.nar`
We shouldn't use specific version for the doc in master branch, right?

Comment (Huanli-Meng) [reply docs/lakehouse-sink.md:208]: yes, I will update it.

File: docs/lakehouse-source.md [hunk 4]
```diff
@@ -74,65 +72,64 @@ Before using the Lakehouse source connector, you need to configure it. This tabl
...
-        "maxReadRowCountOneRound": 100000
+    ```json
+    {
+        "tenant":"public",
+        "namespace":"default",
+        "name":"delta_source",
+        "parallelism":1,
+        "topicName": "delta_source",
+        "processingGuarantees":"ATLEAST_ONCE",
+        "archive": "connectors/pulsar-io-lakehouse-{{connector:version}}.nar",
```
Comment (hangc0276) [inline docs/lakehouse-source.md:118]: For cloud jar, the jar name is `connectors/pulsar-io-lakehouse-{{connector:version}}-cloud.nar`, please help update it, thanks.

Comment (Huanli-Meng) [reply docs/lakehouse-source.md]: Fixed. PTAL

**Review by hangc0276 — APPROVED** [review]

**Pull request merged**